ls1intum/Artemis
# Feature/docker files

### Motivation and Context
Update docker-compose files to enable easier onboarding for new developers.

### Description
The atlassian stack is now configured in a docker-compose setup.
This allows an easier start into the Artemis development.

Also, the Gitlab+Jenkins stack is now further moved to Docker.

### Steps for Testing
Follow the instructions listed [here](https://github.com/ls1intum/Artemis/blob/feature/docker-files/doc/bamboo-bitbucket-jira/BAMBOO_BITBUCKET_JIRA_SETUP.md) and verify that the setup is clearly understandable and  works as intended.
